app.js

```diff
@@ -1,76 +1,82 @@
-const parsers = require("./parsers");
+const kaholoPluginLibrary = require("@kaholo/plugin-library");
+const { default: axios } = require("axios");
+const slack = require("slack");
+
 const autocomplete = require("./autocomplete");
-const SlackService = require("./slack.service");
+const slackListFunctions = require("./slack-list-functions");
 
-async function sendMessageToChannel(action, settings) {
-  const { CHANNEL: channel, group, TEXT: msg } = action.params;
-  const slackService = SlackService.from(action.params, settings);
-  return slackService.sendMessage({
-    channel: parsers.autocomplete(channel),
-    group: parsers.autocomplete(group),
-    msg,
+function sendMessageToChannel({
+  slackToken: token,
+  channel,
+  text,
+}) {
+  return slack.chat.postMessage({
```

Comment:
All the direct `slack` class could go into a common file together with `slack-list-functions` and create a service. That would encapsulate all the Slack related logicwithin file, and hide the implementation details to the `app.js`.